myft/templates/instant-alert.html

```diff
@@ -0,0 +1,34 @@
+{{#if @root.flags.myFtApiWrite}}
+<form class="n-myft-ui n-myft-ui--instant" method="POST"
+	data-myft-ui="instant"
+	data-concept-id="{{conceptId}}"
+	action="/__myft/api/core/followed/concept/{{conceptId}}?method=put">
+	<input type="hidden" value="{{name}}" name="name">
+	<input type="hidden" value="{{taxonomy}}" name="taxonomy">
+	<button
+		aria-label="Get instant alerts for {{name}}"
+		aria-pressed="false"
+		class="n-myft-ui__button
+			{{#variant}} n-myft-ui__button--{{this}}{{/variant}}
+			{{#size}} n-myft-ui__button--{{this}}{{/size}}"
```

Comment:
to match the 'follow' version, you can specify size (follow promo in next-article has this) , and there is an 'inverse' variant which is the dark green colour